### ISSUE TITLE
phpmd ignore SuppressWarnings(PHPMD.UnusedFormalParameter) inside functions

### ISSUE THREAD

**Issue by SailorMax** [phpmd/phpmd#504]
sample code:
``` php
<?
/** @SuppressWarnings(PHPMD.UnusedFormalParameter) */
function innerFunc1($a)
{
}

call_user_func(function() {
	/** @SuppressWarnings(PHPMD.UnusedFormalParameter) */
	function innerFunc2($a)
	{
	}
});
```

Expected: no warnings

Actual: `Avoid unused parameters such as '$a'.` for inlined function.
